src/js/common/schemaform/review/PhoneNumberWidget.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+// When coming from a PhoneNumberWidget (not review), value will be only numbers
+export default function PhoneNumberWidget({ value }) {
+  let formatted = value;
+  if (value && value.length === 10) {
+    formatted = `(${value.substr(0, 3)}) ${value.substr(3, 3)}-${value.substr(6)}`;
+  }
```

Comment:
That's fair; we can always refactor it later if we find we'll be doing this kind of thing for more than these two fields.